wazo_confd/helpers/restful.py

```diff
@@ -75,10 +75,20 @@ def __init__(self, service):
         super().__init__()
         self.service = service
 
+    def unsafe_fallback_get(self, fallback_tenants=None):
+        params = self.search_params()
+        tenant_uuids = self._build_tenant_list(params)
+        if fallback_tenants:
+            tenant_uuids.extend(fallback_tenants)
+
+        return self._get(params, tenant_uuids)
+
```

Comment:
I would scope this code to ingress plugin until someone else need it (to avoid unnecessary indirection)